.github/workflows/build-test-vm.yaml

```diff
@@ -29,6 +29,8 @@ on:
 
 env:
   IMG_POSTGRES_16_BULLSEYE: "neondatabase/vm-postgres-16-bullseye"
+  # using locally build base image
+  IMG_DAEMON: "daemon:dev"
```

Comment:
Asking, so I can understand better: how does daemon:dev gets build in the CI?

Reply:
The makefile has target 
```
docker-build-daemon:
	docker build \
		--tag $(IMG_DAEMON) \
		--file neonvm-daemon/Dockerfile \
``` which is called whenever we buld bin/vm-builder